python-modules/cis_crypto/tests/test_secret.py

```diff
@@ -51,21 +51,23 @@ def test_ssm_provider(self):
         key_material = manager.get_key("fake-access-file-key")
         assert key_material is not None
 
-    @mock_ssm
-    def test_ssm_provider_uuid_salt(self):
-        from cis_crypto import secret
+    # The mock does not work properly here
+    # since this only test getParameter, the test is currently disabled
+    #    @mock_ssm
+    #    def test_ssm_provider_uuid_salt(self):
+    #        from cis_crypto import secret
 
-        os.environ["CIS_SECRET_MANAGER_SSM_UUID_SALT"] = "/salty"
-        client = boto3.client("ssm", region_name="us-west-2")
-        client.put_parameter(
-            Name="/salty",
-            Description="A fake salt",
-            Value="fancyseruresalt",
-            Type="SecureString",
-            KeyId="alias/aws/ssm",
-        )
-        salt = secret.AWSParameterstoreProvider().uuid_salt()
-        assert salt is not None
+    #        os.environ["CIS_SECRET_MANAGER_SSM_UUID_SALT"] = "/salty"
+    #        client = boto3.client("ssm", region_name="us-west-2")
+    #        client.put_parameter(
+    #            Name="/salty",
+    #            Description="A fake salt",
+    #            Value="fancyseruresalt",
+    #            Type="SecureString",
+    #            KeyId="alias/aws/ssm",
+    #        )
+    #        salt = secret.AWSParameterstoreProvider().uuid_salt()
+    #        assert salt is not None
```

Comment:
i could mock uuid salt like i do elsewhere but it makes for a test of no value in this case, so its commented out until mock_ssm does the right thing

Reply:
@andrewkrug